gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/DiskStoreImpl.java

```diff
@@ -407,6 +413,12 @@
    */
   private final DiskBlockSortManager sortManager;
 
+  final long standbyCrfSize;
+  final long standbyDrfSize;
+  final Oplog.OplogFile standByCrf;
```

Comment:
Standbykrf should also be created. Please check and keep space for that too.

Reply:
done. pls review.

Reply:
ok.